frontend/src/main/java/com/java/academy/gym/dto/LocaleUserMessagesRequestDto.java

```diff
@@ -0,0 +1,28 @@
+package com.java.academy.gym.dto;
+
+import java.util.List;
+
+public class LocaleUserMessagesRequestDto {
+
+    private String languageCode;
+    private List<String> messagesKeys;
```

Comment:
It should be map, we want to send this message from client-web to backend and from backend to client-web. 
(in first direction we have empty values)